parquet-column/src/main/java/org/apache/parquet/io/api/RecordConsumer.java

```diff
@@ -125,4 +125,11 @@
    */
   abstract public void addDouble(double value);
 
+  /**
+   * NoOps by default
+   * Subclass class can implement its own flushing logic
+   */
+  public void flush() {
```

Comment:
should this just be called close(), and implement Closeable? Is there a reason for this to not be a Closeable? Or is this called multiple times (close() is for one-time closing)